packages/cli/src/results/project-meta-task-result.ts

```diff
@@ -8,10 +8,20 @@ export default class ProjectMetaTaskResult extends BaseMetaTaskResult implements
   name!: string;
   version!: string;
   repository!: RepositoryInfo;
+  filesForCheckup!: string[];
 
   toConsole() {
+    let filesForCheckupString =
```

Comment:
Can we please not use the type in the variable name?

```suggestion
    let analyzedFilesMessage =
```